### PR TITLE
core(uses-webp): tweak text to be more next-gen focused

### LINKED ISSUE #3557: Rename webp messaging to "next-gen image formats"
On the recent reddit thread some users were flummoxed by the recommendation to adopt and serve webp: https://www.reddit.com/r/webdev/comments/75w7t0/so_exactly_what_do_i_do_google_put_my_css_in_js/doatllq/

Indeed, it is only supported by 1 rendering engine. And unfortunately it's fairly difficult to pull off. 

Given the difficulty and the reduced impact, we should consider removing the audit from the default. WDYT?

### PR DESCRIPTION
**Pull request by patrickhulce**
closes #3557

### CODE REVIEW

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js [hunk 1]
```diff
@@ -21,9 +21,9 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in next-gen formats',
       informative: true,
-      helpText: 'WebP provides better lossy and lossless compression than PNG or JPEG, ' +
+      helpText: 'JP2, JPEG XR, and WebP often provide better compression than PNG or JPEG, ' +
```
Comment (paulirish) [inline lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: let's use the elongated names

> Image formats like JPEG 2000, JPEG XR, and WebP often provide better compression than PNG or JPEG

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js [hunk 2]
```diff
@@ -21,9 +21,9 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in next-gen formats',
```
Comment (paulirish) [inline lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: I'm wondering if there's a better name than "next-gen". The closest i'm getting is "2010-era". But that doesnt' seem worth it.

Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: how about just "better" or "superior"?

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: ya, done

**Comment by paulirish** [comment]
WFM. over to @vinamratasingal as this is user facing

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 1 above)
Comment (addyosmani) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Passing comment: as part of the research we did for images.guide, it turned out that a lot of users could get some pretty hefty wins from adopted MozJPEG. This isn't technically a next-gen format, but a JPEG optimizer. Unsure if you'd want to cater for those types of recommendations.

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js [hunk 3]
```diff
@@ -21,10 +21,10 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in superior formats',
```
Comment (vinamratasingal) [inline lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: "Superior formats" seems a little judgy. Can we just say "next-gen formats" or "performant formats"

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js [hunk 4]
```diff
@@ -21,10 +21,10 @@ class UsesWebPImages extends ByteEfficiencyAudit {
...
       informative: true,
-      helpText: 'WebP provides better lossy and lossless compression than PNG or JPEG, ' +
-        'which means faster downloads and less data consumption. ' +
+      helpText: 'Image formats like JPEG 2000, JPEG XR, and WebP often provide better ' +
+        'compression than PNG or JPEG, which means faster downloads and less data consumption. ' +
         '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/webp).',
```
Comment (vinamratasingal) [inline lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Shouldn't we update this documentation to explain the different formats?

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js [hunk 5]
```diff
@@ -21,10 +21,10 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in superior formats',
       informative: true,
-      helpText: 'WebP provides better lossy and lossless compression than PNG or JPEG, ' +
-        'which means faster downloads and less data consumption. ' +
+      helpText: 'Image formats like JPEG 2000, JPEG XR, and WebP often provide better ' +
```
Comment (vinamratasingal) [inline lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Do you want to mention something here about different browser support for each, or nah?

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 3 above)
Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: 'next-gen' was nixed, but performant SGTM

@paulirish?

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 5 above)
Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Nah we have a policy to keep the details in the docs

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 4 above)
Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Yes, indeed we should

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 3 above)
Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: nevermind paul's good with "next-gen" afterall 👍

File: lighthouse-core/audits/byte-efficiency/uses-webp-images.js (hunk 4 above)
Comment (vinamratasingal) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Filed a bug against Kayce to update the docs for next sprint :)

Comment (patrickhulce) [reply lighthouse-core/audits/byte-efficiency/uses-webp-images.js]: Cool cool, I also just got a brain dump from him on how I can update stuff and seems straightforward when we have changes to docs like this that aren't new material :)

**Comment by benschwarz** [comment]
👍

**Pull request merged**